package.json

```diff
@@ -20,6 +20,7 @@
     "compile:main": "webpack --config webpack.main.ts",
     "compile:renderer": "webpack --config webpack.renderer.ts",
     "compile:i18n": "lingui compile",
+    "compile:extensions": "rollup --config src/extensions/rollup.config.js",
```

Comment:
Let's re-use in line 18 similar to `dev:main`, etc? 
```
"dev:extensions": "yarn compile:extensions --watch"
```

Reply:
Yes, fixed.